threescale_api/auth.py

```diff
@@ -11,6 +11,7 @@ class BaseClientAuth(requests.auth.AuthBase):
     def __init__(self, app, location=None):
         self.app = app
         self.location = location
+        self.credentials = {}
```

Comment:
did pylint report this? otherwise it isnt missing, this is abstract base class (despite to fact it doesnt inherit from abstractclass)

Reply:
Yes, this change is based on pylint report. It is logical because attribute should be defined if it is used in one of methods, see `__call__` of the same class. I understand that this attribute is defined in inherited classes, but also I think that this fix is OK.